src/components/Notes/Notes.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import useStyles from "./Notes.styles";
+import { namespace } from "context";
+import Button from "components/common/Button";
+
+export default function Notes({
+  isMobile,
+  orientation,
+}: {
+  isMobile: boolean;
+  orientation: string;
+}): JSX.Element {
+  const classes = useStyles({ isMobile, orientation });
+  const [notes, setNotes] = React.useState("");
+
+  // load notes if it is in localStorage
+  React.useEffect(() => {
+    const localNotes = localStorage.getItem(`${namespace}notes`);
+    if (localNotes) {
+      setNotes(localNotes);
+    }
+  }, []);
+
+  // data will be saved
+  // when the page loses focus on the textarea
+  // or when the user leaves the page
+  // however, if they do it too soon after the last entry
+  // the notes won't be saved properly (a few chars off)
+  function saveData(notes: string) {
+    localStorage.setItem(`${namespace}notes`, notes);
+  }
```

Comment:
Might want to considered only saving at the end of the session

Reply:
i tried multiple ways to address the note saving feature. Right now it saves `onChange` and `onBlur`

if we want it to save onunload/onbeforeunload, we might have to try something different. in my previous implementation, the last few characters get cut off if I do it too fast. it may be the async nature of `setState`. To combat that, we can do a direct reference on the `textarea`'s value instead, avoiding set state alltogether.

so i guess the question is... do we want to do that? if so, should we continue to have a state to manage the notes?